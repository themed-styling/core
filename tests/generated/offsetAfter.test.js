import 'jest-styled-components'

import { offsetAfter } from '../../src/index.js'

import { testAllOn } from '../utilities.js'

testAllOn(offsetAfter, 'offsetAfter')
