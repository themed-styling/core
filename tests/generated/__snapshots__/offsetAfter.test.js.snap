// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`offsetAfter renders 0 1`] = `
Array [
  .c0 {
  offset-after: 0;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: 0;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: 0;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: 0;
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders function 1`] = `
Array [
  .c0 {
  offset-after: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: has;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: hasnot;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: has;
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders invalid theme path 1`] = `
Array [
  .c0 {
  offset-after: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: invalid.path.1;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: invalid.path.1;
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders negative number 1`] = `
Array [
  .c0 {
  offset-after: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: -54881;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: -42365;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: -42365;
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders object [breakpoints] 1`] = `
Array [
  @media screen and (min-width:November) {
  .c0 {
    offset-after: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    offset-after: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {
  .c0 {
    offset-after: July;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: August;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: September;
  }
}

@media screen and (min-width:August) {
  .c0 {
    offset-after: November;
  }
}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    offset-after: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    offset-after: November;
  }
}

<div
    className="c0"
  />,
  @media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:November) {

}

@media screen and (min-width:August) {

}

@media screen and (min-width:November) {
  .c0 {
    offset-after: May;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: June;
  }
}

@media screen and (min-width:November) {
  .c0 {
    offset-after: April;
  }
}

@media screen and (min-width:August) {
  .c0 {
    offset-after: November;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders object [breakpoints] 2`] = `
Array [
  @media May {
  .c0 {
    offset-after: July;
  }
}

@media April {
  .c0 {
    offset-after: August;
  }
}

@media January {
  .c0 {
    offset-after: September;
  }
}

@media April {
  .c0 {
    offset-after: November;
  }
}

@media June {
  .c0 {
    offset-after: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {
  .c0 {
    offset-after: July;
  }
}

@media April {
  .c0 {
    offset-after: August;
  }
}

@media January {
  .c0 {
    offset-after: September;
  }
}

@media April {
  .c0 {
    offset-after: November;
  }
}

@media June {
  .c0 {
    offset-after: August;
  }
}

@media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    offset-after: January;
  }
}

@media April {
  .c0 {
    offset-after: December;
  }
}

@media January {
  .c0 {
    offset-after: April;
  }
}

@media April {
  .c0 {
    offset-after: May;
  }
}

@media June {
  .c0 {
    offset-after: June;
  }
}

<div
    className="c0"
  />,
  @media May {

}

@media April {

}

@media January {

}

@media April {

}

@media June {

}

@media May {
  .c0 {
    offset-after: January;
  }
}

@media April {
  .c0 {
    offset-after: December;
  }
}

@media January {
  .c0 {
    offset-after: April;
  }
}

@media April {
  .c0 {
    offset-after: May;
  }
}

@media June {
  .c0 {
    offset-after: June;
  }
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders postitive number 1`] = `
Array [
  .c0 {
  offset-after: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: 54881;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: 42365;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: 42365;
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders string 1`] = `
Array [
  .c0 {
  offset-after: July;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: July;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: June;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: June;
}

<div
    className="c0"
  />,
]
`;

exports[`offsetAfter renders valid theme path 1`] = `
Array [
  .c0 {
  offset-after: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: DarkSlateBlue;
}

<div
    className="c0"
  />,
  .c0 {
  offset-after: DarkSlateBlue;
}

<div
    className="c0"
  />,
]
`;
